paper/paper.md

```diff
@@ -177,10 +179,14 @@ p_direction(rnorm(1000, mean = 1, sd = 1))
 
 [**`bayesfactor_savagedickey()`**](https://easystats.github.io/bayestestR/reference/bayesfactor_savagedickey.html)
 computes the ratio between the density of a single value (typically the
-null) in two distributions, typically the posterior vs. the prior
-distributions. This method is used to examine if the hypothesis value is
-less or more likely given the observed data, and is an approximation of a Bayes factor comparing 
-the model against a model in which the parameter of choice is restricted to the point null. 
+null) in two distributions. When these distributions are the prior and the posterior
```

Comment:
@mattansb could you make sure that the docstrings of the function and its appearance in the README are aligned with that (we can hard code the citation there), so that everything stays aligned as much as possible :)

Reply:
DONE!

> we can hard code the citation there

I don't know how to so this...

Reply:
I meant just writing "(Wagenmakers, 2018)" ^^

Reply:
Ah, *that* I can do 😁